common/src/chain/transaction/signature/inputsig/authorize_pubkeyhash_spend.rs

```diff
@@ -69,7 +69,7 @@ pub fn sign_address_spending(
     }
     let msg = sighash.encode();
     let signature = private_key
-        .sign_message(&msg)
+        .sign_message(&msg, randomness::make_true_rng())
```

Comment:
why not pass as an argument?

Reply:
I don't mind.

Reply:
To be clear, in many of these functions I just made a judgment call. I would change my mind if a case is presented. Feel free to change them in your next PR.